ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/productcarousel/v1/productcarousel/productcard.html

```diff
@@ -12,14 +12,13 @@
   ~
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
   <template data-sly-template.productcard="${@ product}">
-      <div role="displaycard"
-           data-product-sku="${product.commerceIdentifier.value}"
+      <div data-product-sku="${product.commerceIdentifier.value}"
            id="${product.id}"
            data-cmp-data-layer="${product.data.json}"
            class="card product__card">
           <a href="${product.URL}">
               <div class="product__card__image">
-                  <img class="product__image" src="${product.imageURL}" alt="${product.imageAlt}" data-gallery-role="currentimage"/>
+                  <img class="product__image" src="${product.imageURL}" alt="${product.imageAlt}" />
```

Comment:
I think we should not remove the `data-gallery-role` because it is not an aria role but, if I am not mistaken, used in the gallery's javascript https://github.com/adobe/aem-core-cif-components/blob/master/ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/product/v1/product/clientlib/js/gallery.js#L187

Reply:
Or maybe it is a copy&paste left over ... as the gallery.js seems unrelated to the product carousel....

Reply:
I checked if it's needed an it's not, so I removed it. I also assume it was just copied over from the product component.